aws/resource_aws_glue_catalog_crawler_test.go

```diff
@@ -0,0 +1,276 @@
+package aws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/glue"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"testing"
+)
+
+func TestAccAWSGlueCrawler_basic(t *testing.T) {
+	const name = "aws_glue_catalog_crawler.test"
+	resource.Test(t, resource.TestCase{
```

Comment:
All the `TestCase` below are missing the `CheckDestroy` function. See other resource acceptance tests and the [Terraform documentation on CheckDestroy](https://www.terraform.io/docs/extend/testing/acceptance-tests/testcase.html#checkdestroy) for more information.

Reply:
Done